os2forms_payment.libraries.yml

```diff
@@ -0,0 +1,8 @@
+os2forms_payment:
+  version: 1.x
+  css:
+    theme:
+      assets/os2forms_payment.css: {}
+  js:
+    assets/os2forms_payment.js: {}
+    https://test.checkout.dibspayment.eu/v1/checkout.js?v=1: {}
```

Comment:
This dependency should be added from code to load the right version and not always the `test` version.

Reply:
Yes. I moved this inclusion into the webform element file.